d3ploy/DemandFac.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
```

Comment:
This module needs a docstring.  Also module names in Python use `lower_case_and_underscores`, so this should be `demand_fac.py`